pex/venv/virtualenv.py

```diff
@@ -250,6 +278,26 @@ def venv_dir(self):
         # type: () -> str
         return self._venv_dir
 
+    @property
+    def _pyvenv_cfg(self):
+        # type: () -> Optional[PyVenvCfg]
+        if not hasattr(self, "__pyvenv_cfg"):
+            self.__pyvenv_cfg = PyVenvCfg.find(self._interpreter.binary)
+        return self.__pyvenv_cfg
+
+    @property
+    def created_by(self):
```

Comment:
The code that writes a `pyvenv.cfg` file above explicitly sets `created_at = ...`, but it seems like this `created_at` method wouldn't reflect that value back out, instead showing `virtualenv ...`; is that expected?

Reply:
It is expected. I did choose a confusing name I guess reading the code, but the name was picked for the reader of the file.

The `created_by`, as with all fields except `home` and `include-system-site-packages`, are arbitrary an un-defined by PEP-405. I just wanted to leave a marker that this was _not_ created by virtualenv to help avoid confusing someone poking about in the venv later. Although modern virtualenv does create a pyvenv.cfg, this ancient single file version Pex vendors does not.

Clearly though I'm just causing different confusion! I did consider emitting a `#`-style comment line with the same content.

Reply:
All good, sounds reasonable. If it really mattered (it doesn't to me), one could use a synonym in the `pyvenv.cfg`, e.g. `creator = ...` or something, unless there's some non-PEP-405 convention that's guiding the choice.

Reply:
No convention. There is in fact no similar field at all in any pyvenv.cfg I've seen. This appears to be unique.

Reply:
I may do something about this, at least comment, in the `pex3 venv create` PR CI burn cycle.